src/lib/events/offlineRsvpSync.ts

```
@@ -0,0 +1,60 @@
+  idempotencyKey: string;
+  queuedAt: number;
+}
+
+const OUTBOX_KEY = 'rsvp_outbox';
+
+export async function getQueuedRsvps(): Promise<QueuedRsvp[]> {
+  try {
+    const queue = await get<QueuedRsvp[]>(OUTBOX_KEY);
+    return queue || [];
+  } catch (error) {
+    console.error('[OfflineRsvpSync] Failed to get queued RSVPs', error);
```

ESLint

[error] Replace `'[OfflineRsvpSync]·Failed·to·get·queued·RSVPs'` with `"[OfflineRsvpSync]·Failed·to·get·queued·RSVPs"`

(prettier/prettier)